model/labels/labels_stringlabels.go

```diff
@@ -283,6 +283,11 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+func (ls Labels) ByteSize() int {
+	return len(ls.data) + int(unsafe.Sizeof(""))
```

Comment:
This is likely good enough:

```suggestion
	return len(ls.data)
```

Reply:
Yes, initially the estimate were also excluding the headers. I will revert to that.